Core/UserDefaultsPropertyWrapper.swift

```diff
@@ -92,9 +92,6 @@ public struct UserDefaultsWrapper<T> {
 
         case shouldScheduleRulesCompilationOnAppLaunch = "com.duckduckgo.ios.shouldScheduleRulesCompilationOnAppLaunch"
 
-        case lastAppTrackingProtectionHistoryFetchTimestamp = "com.duckduckgo.ios.appTrackingProtection.lastTrackerHistoryFetchTimestamp"
-        case appTPUsed = "com.duckduckgo.ios.appTrackingProtection.appTPUsed"
```

Comment:
I haven't added logic to clean up these values as this feature never shipped, but let me know if you'd rather I handle that.

Reply:
Should be fine, I think